tests/e2e/api-core-tests/endpoints/coupons.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Internal dependencies
+ */
+ const { getRequest, postRequest, putRequest, deleteRequest } = require('../utils/request');
+ const { coupon, shared } = require('../data');
+
+/**
+ * WooCommerce Coupon endpoints.
+ *
+ * https://woocommerce.github.io/woocommerce-rest-api-docs/#coupons
+ */
+const couponsApi = {
+	name: 'Coupons',
```

Comment:
Can we add trailing commas to the last item in all of these objects?

Reply:
Done! ✔️